Dockerfile

```
@@ -309,6 +309,10 @@ WORKDIR /rocker-build/
 RUN install2.r --error --deps TRUE \
     multipanelfigure
 
+# Molecular subtyping MB
+RUN R -e "remotes::install_github('d3b-center/medullo-classifier-package', ref = 'master', dependencies = TRUE)"
```

Let's tie the installation of this package from GitHub to a specific commit (using the most recent one, here):

```suggestion
RUN R -e "remotes::install_github('d3b-center/medullo-classifier-package', ref = 'e3d12f64e2e4e00f5ea884f3353eb8c4b612abe8', dependencies = TRUE)"
```